.gitignore

```diff
@@ -1 +1,6 @@
 *.swp
+.kitchen
+.kitchen.local.yml
+.cache
+junit-*.xml
+__pycache__
```

Comment:
Since this file is being touched, do you merging it with the one in `template-formula`?  The only difference is `junit-*.xml`.  Otherwise, the rest can be pulled in from there.